src/main/target/AOCODARCF7DUAL/target.h

```diff
@@ -46,11 +46,11 @@
 #define MPU6000_SPI_BUS         BUS_SPI2
 #define MPU6000_EXTI_PIN        PC4
 
-#define USE_IMU_MPU6500
-#define IMU_MPU6500_ALIGN       CW270_DEG
-#define MPU6500_CS_PIN          PB12
-#define MPU6500_SPI_BUS         BUS_SPI2
-#define MPU6500_EXTI_PIN        PC4
+// #define USE_IMU_MPU6500
```

Comment:
let's do the same here in these other 5 lines

Reply:
Ok, I'll do it right away.

Reply:
good ,you can delete these code.